docs/image.md

```diff
@@ -419,7 +419,7 @@ The image source (either a remote URL or a local file resource).
 
 This prop can also contain several remote URLs, specified together with their width and height and potentially with scale/other URI arguments. The native side will then choose the best `uri` to display based on the measured size of the image container. A `cache` property can be added to control how networked request interacts with the local cache. (For more information see [Cache Control for Images](images#cache-control-ios-only)).
 
-The currently supported formats are `png`, `jpg`, `jpeg`, `bmp`, `gif`, `webp`, `psd` (iOS only). In addition, iOS supports several RAW image formats. Refer to Apple's documentation for the current list of supported camera models (for iOS 12, see https://support.apple.com/en-ca/HT208967).
+The currently supported formats are `png`, `jpg`, `jpeg`, `bmp`, `gif`,`webp` (Android only), `psd` (iOS only). In addition, iOS supports several RAW image formats.`webp` format is supported on iOS only when bundled with the JavaScript code. Refer to Apple's documentation for the current list of supported camera models (for iOS 12, see https://support.apple.com/en-ca/HT208967).
```

Comment:
> `webp` format is supported on iOS only when bundled with the JavaScript code

Where is this coming from? Do you have a code reference from React Native or a GitHub issue for reference?

Reply:
@cortinico 
Github Issue is opened.
https://github.com/facebook/react-native-website/issues/4165

Reply:
```suggestion
The currently supported formats are `png`, `jpg`, `jpeg`, `bmp`, `gif`,`webp`, `psd` (iOS only). In addition, iOS supports several RAW image formats. Refer to Apple's documentation for the current list of supported camera models (for iOS 12, see https://support.apple.com/en-ca/HT208967).
```

Reply:
And let's add a separate paragraph about webp below please

Reply:
Sure @cortinico

Reply:
@cortinico added as a paragraph. Please take a look. Thanks